lib/browser/api/app.ts

```diff
@@ -17,6 +17,29 @@ let dockMenu: Electron.Menu | null = null
 Object.setPrototypeOf(App.prototype, EventEmitter.prototype)
 EventEmitter.call(app as any)
 
+// Properties.
+
+const nativeASGetter = app.isAccessibilitySupportEnabled
+const nativeASSetter = app.setAccessibilitySupportEnabled
+Object.defineProperty(App.prototype, 'accessibilitySupportEnabled', {
+  get: () => nativeASGetter.call(app),
```

Comment:
Maybe use `get: function () { this.isAccessibilitySupportEnabled() }` or 
`get: () => app.isAccessibilitySupportEnabled()`?

One benefit is, if user monkey-patches the method the property would also be patched automatically.